projects/adrv9026/common/adrv9026_bd.tcl

```diff
@@ -215,7 +215,7 @@ ad_cpu_interconnect 0x7c400000 axi_adrv9026_rx_dma
 
 # gt uses hp0, and 100MHz clock for both DRP and AXI4
 
-ad_mem_hp0_interconnect $sys_cpu_clk sys_ps8/S_AXI_HP0
+ad_mem_hp0_interconnect $sys_cpu_clk sys_ps7/S_AXI_HP0
```

Comment:
Won't this break the ZCU102 project?

Reply:
Tested and confirmed that it doesn't break it